kotlinx-coroutines-test/common/test/Helpers.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
It looks like you should update your licensing template in IDEA, the year should be 2021

Reply:
Dunno, I checked, and the configuration seems alright.